explorer/nodewatch/tests/test_app.py

```diff
@@ -184,12 +184,106 @@ def test_get_api_symbol_nodes_peer(client):  # pylint: disable=redefined-outer-n
 	# Assert: spot check names
 	assert 200 == response.status_code
 	assert 'application/json' == response.headers['Content-Type']
-	assert 5 == len(response_json)
+	assert 7 == len(response_json)
 	assert [
-		'Apple', 'Shin-Kuma-Node', 'ibone74', 'jaguar', 'symbol.ooo maxUnlockedAccounts:100'
+		'Apple', 'Shin-Kuma-Node', 'ibone74', 'jaguar', 'symbol.ooo maxUnlockedAccounts:100', 'xym pool', 'yasmine farm'
 	] == list(map(lambda descriptor: descriptor['name'], response_json))
 
 
+def test_get_api_symbol_nodes(client):  # pylint: disable=redefined-outer-name
+	# Act:
+	response = client.get('/api/symbol/nodes')
```

Comment:
do we need a new /nodes route in addition to nodes/peer and nodes/api?

Reply:
I was thinking about this as well, but not sure it's a good idea, here is my thought.

purpose
- the endpoint mainly returns all nodes and displays them on explorer.
- this endpoint added SSL, limit, and order filters, which will use to init explorer, and wallet to pull all healthy nodes.

calling 1 request instead of 2 to get all node lists.

But I didn't think that is the problem called 2 requests to get the node list, I can apply query params in the 2 endpoints as well.

what will you recommand?

Reply:
> I can apply query params in the 2 endpoints as well

I think I would prefer this approach, but also ask @gimre-xymcity his opinion.